src/EventStore.Core/Data/Redaction/GetEventPositionResult.cs

```diff
@@ -0,0 +1,15 @@
+namespace EventStore.Core.Data.Redaction {
+	public enum GetEventPositionResult {
+		Success = 0,
```

Comment:
Probably worth having a 'None' as the zero value to avoid mistaking uninitialized for success at some point. Same for the other enums in the same folder

Reply:
fixed :+1: